code/modules/awaymissions/gateway.dm

```diff
@@ -2,106 +2,243 @@ GLOBAL_DATUM(the_gateway, /obj/machinery/gateway/centerstation)
 
 /obj/machinery/gateway
 	name = "gateway"
-	desc = "A mysterious gateway built by unknown hands, it allows for faster than light travel to far-flung locations."
+	desc = "A gateway built for quick travel between linked destinations."
 	icon = 'icons/obj/machines/gateway.dmi'
 	icon_state = "off"
 	density = TRUE
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
-	var/active = 0
+	move_resist = INFINITY
+	var/active = FALSE
 	var/checkparts = TRUE
-	var/list/obj/effect/landmark/randomspawns = list()
-	var/calibrated = TRUE
-	var/list/linked = list()
-	var/can_link = FALSE	//Is this the centerpiece?
+	var/list/adjacent_parts = list()
+	var/centerpiece = FALSE	//Is this the centerpiece?
```

Comment:
I'd suggest changing it so instead of being multiple gateways machines its one big machine with filler like DNA vault for example

Reply:
that would require me to figure out somehow how to merge the 3x3 icons

Reply:
its fine then

Reply:
although i see TG already did that 

![image](https://github.com/user-attachments/assets/7d865b10-eafa-472b-ac5d-5d31dc7aab99)

Reply:
Implemented, well one part of it at least.